airflow/cli/cli_parser.py

```diff
@@ -433,6 +433,12 @@ def string_lower_type(val):
 ARG_RUN_ID = Arg(("-r", "--run-id"), help="Helps to identify this run")
 ARG_CONF = Arg(("-c", "--conf"), help="JSON string that gets pickled into the DagRun's conf attribute")
 ARG_EXEC_DATE = Arg(("-e", "--exec-date"), help="The execution date of the DAG", type=parsedate)
+ARG_REPLACE_MICRO = Arg(
+    ("--replace-microseconds",),
+    help="whether microseconds should be zeroed",
+    action="store_false",
+    default=True,
+)
```

Comment:
The looks wrong…? If I understand the structure correctly, this would make

```
airflow trigger --replace-microseconds
```

NOT zero-out microseconds. I think the correct implementation is

```suggestion
ARG_REPLACE_MICRO = Arg(
    ("--no-replace-microseconds",),
    help="whether microseconds should be zeroed",
    dest="replace_microseconds",
    action="store_false",
    default=True,
)
```